README.md

```diff
@@ -42,9 +47,10 @@ that have hardened isolation and can run almost any workload that runs in VMs.
 Sysbox does this by making the container resemble a VM-like environment as much
 as possible, using advanced OS virtualization techniques.
 
-Unlike alternative runtimes such as Kata and KubeVirt, **it does not use VMs**;
+Unlike alternative runtimes such as Kata and KubeVirt, **it does not use VMs**.
 this makes it easier to use (particularly in cloud environments by avoiding
-nested virtualization). See [here](#comparison-to-related-technologies) for a
+nested virtualization), although it does not provide the level of isolation
+that VM-based runtimes do. See [here](#comparison-to-related-technologies) for a
```

Comment:
s/does not provide/does not (yet) provide/ :-)

Reply:
Done.